src/main.rs

```diff
@@ -64,6 +64,8 @@ fn main() {
     let mut path_mapping_file = "";
     let mut filter_option = None;
     let mut num_threads = num_cpus::get() * 2;
+    let mut output_file_path = "-";
```

Comment:
I'd make this an option, None by default and in output.rs then you can use stdio when this is None.

Reply:
I made it an option (None by default) in the newest commit.